go.mod

```diff
@@ -14,3 +14,5 @@ require (
 	knative.dev/networking v0.0.0-20220505013701-91dcb36802a9
 	knative.dev/pkg v0.0.0-20220505013700-a8b7d99374a3
 )
+
+replace knative.dev/networking => github.com/skonto/networking v0.0.0-20220511230850-2595d6d1c7c0
```

Comment:
This should be removed when the PR here is merged: https://github.com/knative/networking/pull/668